package.json

```diff
@@ -7,7 +7,26 @@
     "monorepo",
     "sui"
   ],
-  "scripts": {},
+  "scripts": {
+    "commit": "sui-mono commit",
```

Comment:
We are using "co" everywhere. Maybe keep it as co? or standarize this everywhere :\

I personally do npm run co and if don't works I assume it does not have cz and use git commit.